csharp/src/Ice/ITransceiver.cs

```diff
@@ -21,13 +21,15 @@ public interface ITransceiver
         /// <param name="monitor">The ACM monitor object.</param>
         /// <param name="connector">The connector associated with the new connection, this is always null for incoming
         /// connections.</param>
+        /// <param name="connectionId">The connection ID associated with the new connection.</param>
```

Comment:
Should we mention that is always empty for incoming connections?

Reply:
Yes, fixed!